Implemented tasks from README.md

DEz-1996 opened the pull request:
Created classes Truck, Bulldozer,
Excavator with override methods doWork() and stopWork()
Created array Machine[] machines
Created loop foreach to call methods doWork() and stopWork() from array Machine[] machines

d1sam commented on src/main/java/core/basesyntax/MainApp.java:
```diff
@@ -1,5 +1,15 @@
 
 public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
+        Truck truck = new Truck();
+        Machine[] machines = new Machine[]{excavator, bulldozer, truck};
```
Make a note that you can call new Excavator(), new Bulldozer(), new Truck() explicitly in array initialization without temporary variables.

DEz-1996 replied:
> Make a note that you can call new Excavator(), new Bulldozer(), new Truck() explicitly in array initialization without temporary variables.

Thank you for your advice!
Added a new commit:
Moved call new Excavator(), new Bulldozer(), new Truck() in array initialization Machine[] machines in class MainApp

d1sam reviewed (changes requested):
Good job, but there is one point to refactor. Please see my comment below.

DEz-1996 commented:
Thank you for your advice!
Added a new commit:
Moved call new Excavator(), new Bulldozer(), new Truck() in array initialization Machine[] machines in class MainApp

kshuryhin reviewed (approved):
Good job!
